src/dmd/frontend.d

```diff
@@ -54,3 +62,161 @@ void initDMD()
     builtin_init();
 }
 
+/**
+Add import paths to the `global.path`.
+Params:
+    path = range of imports to add
+*/
+void addImports(T)(T path)
+{
+    import dmd.globals : global;
+    import dmd.arraytypes : Strings;
+
+    Strings* res = new Strings();
+    foreach (p; path)
+    {
+        import std.string : toStringz;
+        Strings* a = new Strings();
```

Comment:
Why is an array created for each path, why not added the path directly to `res`?